addons/project/static/src/services/project_task_recurrence.js

```diff
@@ -12,92 +12,42 @@ class ProjectTaskRecurrence {
         this.resModel = 'project.task';
     }
 
-    async stopRecurrence(tasks, callback = () => {}) {
-        const taskIdsWithRecurrence = [];
-        const tasksPerRecurrence = {};
-        const recurrenceIds = [];
+    async addressRecurrence(tasks, mode, callback = () => {}) {
+        const recurrentTaskIds = [];
+        var hasRecurrentTasks = false;
```

Comment:
Use `let` instead of `var`
```suggestion
        let hasRecurrentTasks = false;
```